packages/manager/modules/pci/src/projects/project/streams/onboarding/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_streams_title": "ioStream",
+  "pci_projects_project_streams_onboarding_content1": "Vous n'avez pas encore créé d'IO Stream.",
+  "pci_projects_project_streams_onboarding_content2": "ioStream vous permet de rendre vos architectures orientées services ou microservices plus fiables et évolutives.",
```

Comment:
Sometimes `io` is in Uppercase sometimes lowercase, any reason ?

Reply:
Seems fixed ?? https://github.com/ovh-ux/manager/blob/feat%2Fstream/packages/manager/modules/pci/src/projects/project/streams/onboarding/translations/Messages_fr_FR.json